go.mod

```diff
@@ -4,26 +4,26 @@ go 1.15
 
 require (
 	github.com/confluentinc/bincover v0.1.0
-	github.com/cosmos/cosmos-sdk v0.40.0-rc3
+	github.com/cosmos/cosmos-sdk v0.40.0-rc3.0.20201116225759-10d5ce39adf1
 	github.com/cosmos/ledger-go v0.9.2 // indirect
 	github.com/gogo/protobuf v1.3.1
 	github.com/golang/protobuf v1.4.3
 	github.com/gorilla/mux v1.8.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/imdario/mergo v0.3.11
-	github.com/pkg/errors v0.9.1
 	github.com/rakyll/statik v0.1.7
 	github.com/spf13/cast v1.3.1
 	github.com/spf13/cobra v1.1.1
-	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.6.1
 	github.com/tendermint/tendermint v0.34.0-rc6
-	github.com/tendermint/tm-db v0.6.2
-	google.golang.org/genproto v0.0.0-20201014134559-03b6142f0dc9
-	google.golang.org/grpc v1.33.1
+	github.com/tendermint/tm-db v0.6.3
+	google.golang.org/genproto v0.0.0-20201111145450-ac7456db90a6
+	google.golang.org/grpc v1.33.2
 
 )
 
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.2-alpha.regen.4
 
 replace github.com/cosmos/ledger-cosmos-go => github.com/crypto-com/ledger-cosmos-go v0.9.10-0.20200929055312-01e1d341de0f
+
+replace github.com/cosmos/cosmos-sdk => github.com/yihuang/cosmos-sdk v0.34.4-0.20201117081135-e71014e005c0
```

Comment:
what is this custom fork -- https://github.com/yihuang/cosmos-sdk/commit/e71014e005c0 ? we'll ideally use the released versions.
the coin parsing may not make it to a release.
I can potentially cherry-pick the necessary commits over "v0.40.*" or "release/v0.40" (when it exists), but it'll ideally be some finalised commit that made it to master

Reply:
the changed ParseCoins is already merged into master, but I find the ParseCoin also need to change, hence a second PR. We can wait for it to be merged.

Reply:
upstream pr is merged now.